### DISCUSSION TITLE
Compatibility with scVI corrected data

### DISCUSSION THREAD

**Discussion by jesswhitts** [aertslab/SCENIC#415] [* General SCENIC questions]
Hello, 

I have a large scVI integrated dataset and was wondering what the best way to implement scenic is?

Should I use the full raw counts matrix, or the HVG-resitricted scVI normalised matrix?

Unfortunately, it seems there is no way to retrieve a batch corrected matrix that is not normalised from scVI

Many thanks,
jess

**Comment by jesswhitts** [comment]
I have an update on this - I manage to get a fully corrected expression matrix from scVI. But, the outputs of using the raw data and the batch corrected data are quite different. 

Is it appropriate to use batch corrected counts for pyscenic?

**Reply by johnblake1** [reply]
Hello! Probably off topic of your initial question but still, 

I have been puzzling over the problem of how to retrieve fully corrected count matrix from scvi for quite some time now, so if you could share how did you manage to do this (ideally with the code example) I would be very grateful! 

John

**Reply by jesswhitts** [reply]
No worries - please see my post on the scverse page here

https://discourse.scverse.org/t/compatibility-between-scvi-and-scenic/1795
